src/site/content/en/blog/sms-receiver-announce/index.md

```diff
@@ -0,0 +1,401 @@
+---
+title: Easy OTP on the web with the SMS Receiver API
+subhead: Help users type OTP received through SMS
+authors:
+  - agektmr
+
+date: 2019-09-19
+# updated: 2019-06-27
+
+hero: hero.png
+# You can adjust the fit of your hero image with this property.
+# Values: contain | cover (default)
+# hero_fit: contain
+# You can adjust the position of your hero image with this property.
+# Values: top | bottom | center (default)
+# hero_position: bottom
+# You can provide an optional cropping of your hero image to be used as a
+# thumbnail. Note the alt text will be the same for both the thumbnail and
+# the hero.
+# thumbnail: thumbnail.jpg
+alt: A description of the hero image for screen reader users.
+
+description: |
+  Finding, memorizing and typing OTP sent via SMS to a form is cumbersome. SMS
+  Receiver API will help users typing the OTP.
```

Comment:
```suggestion
  Receiver API makes this process as easy as one tap for the user.
```

This is how you describe it later on in the post. I think this phrasing captures the idea better.